tests/models/llama/test_llama.py

```diff
@@ -14,12 +13,22 @@ def test_llama(record_property):
     tokenizer = AutoTokenizer.from_pretrained(
         model_name, padding_side="left", torch_dtype=torch.bfloat16
     )
+    tokenizer.pad_token = tokenizer.eos_token
```

Comment:
I bet this can be removed

Reply:
Unfortunately, it seems like if I want to pad the inputs, Llama will emit this error if I don't specify a pad_token. 

```
ValueError: Asking to pad but the tokenizer does not have a padding token. Please select a token to use as `pad_token` `(tokenizer.pad_token = tokenizer.eos_token e.g.)` or add a new pad token via `tokenizer.add_special_tokens({'pad_token': '[PA...
```